### ISSUE TITLE
Deprecate Challenge DF in a branch. (Merge after Nov 30)

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/df-web#671]
This is the df-web side of EPIC "wind down Challenge DF" [df-private#63](https://github.com/oceanprotocol/df-private/issues/63).

Do it in a branch for now. Then merge to main the day when Challenge DF ends. (Approx Nov 30, but we have to time it right with the different moving parts.)

**Comment by KatunaNorbert** [comment]
Should we remove the entire Challenge-DF page or just the substream?

**Comment by trentmc** [comment]
> Users might still want to check the winners from the last round so It makes sense to keep it for at least 1 week, but after that we could just remove it

Good point. So let's keep it as-is for this week, and then next week wholly remove the page.

**Comment by KatunaNorbert** [comment]
Good. I will remove the substream and do some small tweaks to Challenge-DF page so it points out that round 66 was the last one. Will provide an screenshot soon.

**Comment by KatunaNorbert** [comment]
<img width="1585" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOGd5QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9afb03e0610147edc46a55ee41e92f81c2ebcda6/Screenshot%202023-12-05%20at%2016.18.26.png" alt="Screenshot 2023-12-05 at 16.18.26.png" />
